lib/recurly/risk/three-d-secure/strategy/cybersource.js

```diff
@@ -0,0 +1,101 @@
+import errors from '../../../errors';
+import Promise from 'promise';
+import ThreeDSecureStrategy from './strategy';
+import { Frame } from '../../../frame';
+
+const debug = require('debug')('recurly:risk:three-d-secure:cybersource');
+
+export default class CybersourceStrategy extends ThreeDSecureStrategy {
+  static strategyName = 'cybersource';
+  static PREFLIGHT_TIMEOUT = 30000;
```

Comment:
Let's move the `PREFLIGHT_TIMEOUT` to `ThreeDSecureStrategy` since the `WorldpayStrategy` uses the same timeout value.